### PR TITLE
Fix #13 Add OSSPH Logo

### LINKED ISSUE #13: Add OSSPH Logo
Add OSSPH logo to the new index page - https://ossphilippines.github.io/freefolio/

Use the [Press Kit](https://drive.google.com/drive/u/2/folders/1uVuUZmfQrIh-vs17rK5soPS05wuS4vkQ)

### PR DESCRIPTION
**Pull request by redKath**
Add OSSPH logo to the new index page
I used the 4.png under ossph-logo-text from the [Press Kit](https://drive.google.com/drive/u/2/folders/1uVuUZmfQrIh-vs17rK5soPS05wuS4vkQ).

### CODE REVIEW

**Comment by jofftiquez** [comment]
Thanks @redKath may I see the outcome? Please send a screenshot here. Thanks.

File: index.html [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
 
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Freefolio</title>
-    <link rel="shortcut icon" href="./clean//img/logo.svg" type="image/x-icon">
-    <script src="//cdn.tailwindcss.com"></script>
-</head>
+    <head>
```
Comment (jofftiquez) [inline index.html:4]: Is this 2 spaces or 4 spaces? Use 2 spaces for tabs please, thanks.

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (redKath) [reply index.html]: Here is the preview of the outcome.
Screenshot 2022-10-03 at 10-29-11 Freefolio
I'll create a new PR for the 2 spaces for the tabs.

**Review by jofftiquez — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (jofftiquez) [reply index.html]: Nice! Thank you!

**Pull request merged**